packages/core/test/server.test.js

```diff
@@ -70,6 +72,21 @@ describe('Snapshot Server', () => {
     await expect(response.text()).resolves.toBe(bundle);
   });
 
+  it('serves the legacy percy-agent.js dom bundle', async () => {
+    let bundle = require('fs')
+      .readFileSync(require.resolve('@percy/dom'), { encoding: 'utf-8' })
+      .concat('(window.PercyAgent = class PercyAgent { snapshot(n, o) { return PercyDOM.serialize(o) } });');
+
+    percy.loglevel('warn');
+    await percy.start();
```

Comment:
Just in case warnings ever come from start for whatever reason, these should be flipped to make sure we're only warning what we're expecting.
```suggestion
    await percy.start();
    percy.loglevel('warn');
```